tests/test_score_function.py

```diff
@@ -92,7 +92,7 @@ def test_score_rescale(self):
             (F - torch.tensor([0.901383399963379, 0.871010780334473, 0.503565192222595])).abs_().max() < EPS
         )
 
-    def test_idf_score_rescale(self):
```

Comment:
Can you keep the old `test_idf_score_rescale` so that we can detect if only the rescaling is not working?